### ISSUE TITLE
Screen reader issues

### ISSUE THREAD

**Issue by IeuanWalker** [AndreiMisiukevich/CardView#385]
Hi,

The screen reader on iOS _(haven't tested android)_ doesn't stay on the current slide and starts readying stuff from the other slides.

Attached is a vid of what is happening.
https://user-images.githubusercontent.com/6544051/116270371-ed0d8d80-a776-11eb-8d5f-49f18d4f213e.MOV

**Comment by AndreiMisiukevich** [comment]
Hey, I don't know how to help with that. This is a very tricky.

You can try to set IsEnabled to False for none selected views.

**Comment by AndreiMisiukevich** [comment]
Or you even can make next/prev items invisible (don't forget to make them visible again before navigating)

**Comment by AndreiMisiukevich** [comment]
Or this approach

```csharp
var b = new Button();
b.SetValue(AutomationProperties.IsInAccessibleTreeProperty, false); // False to remove from accessibility tree or True to add
```

it will help to remove unnecessary elements from the accessibility tree.

I mean make current view's elements accessible and make unaccessible all other elements

**Comment by AndreiMisiukevich** [comment]
I propose next approach

1) In your data template view make all elements `IsInAccessibleTreeProperty = false`
2) Subscribe ItemDisappearing event and perform setting  `IsInAccessibleTreeProperty = false` to all elements as well (Work with CurrentView)
3) Subscribe ItemAppeared and perfor setting `IsInAccessibleTreeProperty = true` to all elements on CurrentView


if you wish, I can add a new event "CurrentViewChanged" for easier implementing this approach. But unfortunately you will have to do it on your side, because side views are added to screen for soother animation.

**Comment by AndreiMisiukevich** [comment]
I implemented something.
Please try 2.7.7

**Comment by IeuanWalker** [comment]
@AndreiMisiukevich Your fix worked :D thanks so much

**Comment by AndreiMisiukevich** [comment]
> @AndreiMisiukevich Your fix worked :D thanks so much

Then need to implement something similar for Android

**Comment by AndreiMisiukevich** [comment]
Need to do the same, but set importantForAccessibility = no-hide-descendants

**Comment by AndreiMisiukevich** [comment]
Implemented for android in 2.7.8

**Comment by IeuanWalker** [comment]
@AndreiMisiukevich Great thanks. 

I haven't started looking at android accessibility atm.
But will let you know if there are any issues :)
